scripts/train_all.sh

```diff
@@ -0,0 +1,5 @@
+# Useage: scripts/train_all.sh drop-only
```

Comment:
rename file to be clear we're using slurm
also typo in usage

Reply:
Done